src/entity/pod.go

```diff
@@ -18,17 +18,24 @@ type Container struct {
 	Command []string `bson:"command" json:"command" validate:"required,dive,required"`
 }
 
+// PodRoute is the structure for add IP routing table
+type PodRoute struct {
+	DstCIDR string `bson:"dstCIDR" json:"dstCIDR" validate:"required,cidr"`
+	Gateway string `bson:"gateway" json:"gateway" validate:"min=1,max=4095"`
```

Comment:
min,max validate for string gateway ?

Reply:
Sorry, My bad. Fix soon.

Reply:
max=4095,min=0

Reply:
Vlan part, update here linkernetworks/network-controller#67

Gateway use ip validate, this part was updated in this PR.